packages/lit-dev-content/site/docs/v3/libraries/labs.md

```diff
@@ -65,6 +65,20 @@ A package containing controllers and decorators for using the [Context Protocol]
 <tr>
 <td>
 
+[task](https://www.npmjs.com/package/@lit-labs/task)
+
+</td>
+<td>A reactive controller for handling asynchronous tasks.</td>
+<td class="labs-table-links">
+
+[📄&nbsp;Docs](/docs/v3/data/task/ "Docs")<br>[💬&nbsp;Feedback](https://github.com/lit/lit/discussions/3361 "Feedback")<br>[🐞&nbsp;Issues](https://github.com/lit/lit/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+%5Blabs%2Ftask%5D "Issues")
+
+</td>
+</tr>
```

Comment:
I think this can be removed? I don't think we are tracking the graduated packages here.